acestep/audio_utils.py

```
@@ -269,14 +265,21 @@ def save_audio(
-                    channels_first=True,
-                    backend='ffmpeg',
-                )
+                # Opus and AAC: encode via ffmpeg subprocess (torchaudio ffmpeg backend requires torchcodec)
+                import soundfile as sf
+                with tempfile.NamedTemporaryFile(suffix=".wav", delete=False) as _tmp:
+                    _tmp_wav = Path(_tmp.name)
+                try:
+                    audio_np = audio_tensor.transpose(0, 1).numpy()
+                    sf.write(str(_tmp_wav), audio_np, sample_rate)
+                    _codec = "libopus" if format == "opus" else "aac"
+                    subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)